Cognite.Simulator.Tests/ExtensionsTests/DataPointsTest.cs

```diff
@@ -49,13 +49,15 @@ public async Task TestGetSample()
             var cdf = provider.GetRequiredService<Client>();
             var dataPoints = cdf.DataPoints;
 
+            await SeedData.GetOrCreateTestTimeseries(cdf).ConfigureAwait(false);
+
             // Tests assume the given time series exists in the test project and
             // that it contains data points in this time range
             var samplingConfiguration = new SamplingConfiguration(start: 1631294940000, end: 1631296740000);
             
             // Test max aggregation. A single data point should be returned
             var (timestamps, values) = await dataPoints.GetSample(
-                "SimConnect-IntegrationTests-OnOffValues",
+                "utils-tests-OnOffValues",
```

Comment:
this is just a rename so we don't mix the test cases with the actual `SimConnect`